docs/solution-architecture/accessControlPolicies.md

```diff
@@ -86,7 +86,7 @@ various resources on which a user can perform actions:
 | **_Document rechten_**                        |                                                                                      |             |               |               |                                                                  |
 | lezen                                         |                                          X                                           |      X      |       X       |               | OPA policy `document lezen`                                      |
 | wijzigen                                      | X (zaak open, document onvergrendeld, niet definitief of vergrendeld door gebruiker) |             |       X       |               | OPA policy `document wijzigen`                                   |
-| verwijderen                                   |                                          X                                           |             |       X       |               | OPA policy `document verwijderen`                                |
+| verwijderen                                   |                      X (Document onvergrendeld door gebruiker)                       |             |       X       |               | OPA policy `document verwijderen`                                |
```

Comment:
What does `onvergrendeld door gebruiker` mean? I would think `onvergrendeld` is enough or not? Does it really matter that it was the current user who did the unlocking?

Reply:
This is also what `document.vergrendeld == false` means: simply: `document onvergrendeld`

Reply:
I followed the convention of the rest of the document, but I agree we could make it onvergrendeld

Reply:
Well, e.g. `vergrendeld door gebruiker` really means: locked _by the current user trying to change this document_`. Whereas `onvergrendeld` simply means: unlocked, no matter by whom. So it is really different.